### PR TITLE
Make HACKING.md bootstrapping instructions more explicit

### PR DESCRIPTION
**Pull request by shonfeder**
I managed to forget how to run these three steps in the weeks since I last hacked on dune. This will help future me, and probably one or two others approaching the project for the first time.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ In order to build itself, Dune uses an OCaml script
 single `boot.ml` file. This file is built using `ocamlopt` or `ocamlc`
 and used to build everything else.
 
+To get the development version bootstraped and installed, run the following
+while in the project's root directory:
+
+```sh
+$ make boot.exe # create the dune.exe file
+$ ./boot.exe    # bootstrap the dune build
```
Comment (emillon) [inline HACKING.md]: I think that the two first commands can be replaced by just `make`. Is there anything missing when you run this?

Comment (shonfeder) [reply HACKING.md]: Oh yes, of course! I'll set up a clean build to be certain, but I'm sure you're right.

**Comment by shonfeder** [comment]
Comment addressed.

File: HACKING.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ In order to build itself, Dune uses an OCaml script
 single `boot.ml` file. This file is built using `ocamlopt` or `ocamlc`
 and used to build everything else.
 
+To get the development version bootstraped and installed, run the following
```
Comment (zploskey) [inline HACKING.md]: Should be "bootstrapped".

**Comment by emillon** [comment]
Thanks!

**Pull request merged**